src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/KerrSchild.hpp

```diff
@@ -214,6 +214,28 @@ namespace Solutions {
  *
  * Right now we use (\f$\ref{eq:sphertocartsimple}\f$), but we may
  * wish to use the other transformation in the future.
+ *
+ * ## Boost of the Kerr-Schild solution
+ *
+ * We add initial momentum to the solution by applying a Lorentz boost to the
+ * metric. Since the Kerr-Schild metric can be expressed covariantly in terms of
+ * the Minkowski metric, a scalar function and a one form, we constructing the
+ * metric by boosting the these objects individually. Notice that we also need
+ * to appropriately boost the coordinates to the boosted frame.
+ *
+ * \warning While technically the boosted Kerr-Schild metric is dependent on
+ * both the time and space coordinates, we have implemented it only at $t = 0$
+ * as in SpEC. Therefore it is technically not an analytic solution and should
+ * not be used to compute errors with respect to it.
```

Comment:
The solution should be stationary in the boosted frame; so shouldn't all time derivatives be zero and this be a solution at any $t$? Am I misunderstanding something?

Reply:
I believe it is stationary in the rest frame, which is not the frame we want. In the frame we want, the black hole is moving and so the metric will be dependent on t and x. I didn't code this t dependence  since I didn't see this in SpEC --it seems to me it should only enter in the function that computes the radius to the center. 

I also didn't code the time derivatives of the lapse and shift in the frame where the black hole is moving (which are zero in the rest frame of course) but left them as zero. This should be fine because I think they are replaced by the gauge functions we use to evolve this.

Reply:
Ok I think you're correct. Let me look at the SpEC implementation too.